librz/core/cmd_analysis.c

```diff
@@ -9340,16 +9340,16 @@ static int cmd_analysis_all(RzCore *core, const char *input) {
 	case 'c': // "aac"
 		switch (input[1]) {
 		case '*': // "aac*"
-			cmd_analysis_calls(core, input + 1, true, false);
+			rz_cmd_analysis_calls (core, input + 1, true, false);
```

Comment:
make sure to run `git-clang-format` to respect the clang-format style. In particular, no more spaces before function calls.